xapi_bridge/exceptions.py

```diff
@@ -32,7 +32,7 @@ def update_sentry_scope(self, scope, level='warning', user=None, **kwargs):
         if user:
             scope.user = {"username": user}  # TODO: get email if possible
         if kwargs:
-            for key, value in kwargs.iteritems():
+            for key, value in list(kwargs.items()):
```

Comment:
Why create a list instead of just `for key, value in kwargs.items():` ?

Reply:
@johnbaldwin this appears to be an automatic conversion done by `2to3`.